modules/accredited_representative_portal/app/services/accredited_representative_portal/rep_user_creator.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class RepUserCreator < SignIn::UserCreator
```

Comment:
I would also ask if we can unravel this and copy the functions from UserCreator into this function, but we might just remove the concept of `UserCreator` altogether, so maybe not a big deal